app/frontend/styles/application.scss

```diff
@@ -6,6 +6,7 @@
   @return url("~assets/images/" + $filename);
 }
 
+$app-grey: #ebf2f6;
```

Comment:
Where do we put this?

Reply:
what was the decision to move it into here, rather than leaving the hex in `_grid.scss`?

Reply:
We're using it in the [itt_grid.scss](https://github.com/DFE-Digital/apply-for-teacher-training/pull/8784/files/3cbc04559a0590a2bf6beabcecd85a5058041a9c#diff-1151618e52e9d455283ecfc9b6b3907e411fb4bea7dce7e623c522a7fbd34152R12) aswell

Reply:
Ah, of course! – I'm happy to live with it in here for now, as there isn't a more preferable answer coming to me, based on our current directory structure?